restapi/src/main/java/io/stargate/web/docsapi/service/query/search/db/impl/FilterPathSearchQueryBuilder.java

```diff
@@ -76,7 +77,7 @@ public FilterPath getFilterPath() {
     if (parentSize >= DocumentDB.MAX_DEPTH) {
       throw new ErrorCodeRuntimeException(ErrorCode.DOCS_API_GENERAL_DEPTH_EXCEEDED);
     } else {
-      String field = filterPath.getField();
+      String field = DocsApiUtils.convertEscapedCharacters(filterPath.getField());
```

Comment:
I would strongly disagree to do any conversions in the QueryBuilders. The unescaped stuff should already be in the in the path, no need to change here..

Reply:
Can be resolved.